simple_interest.py

```diff
@@ -1,18 +1,32 @@
+
+#just testing my branching first
+
+#just teting my branching first
+
+
 #In this exercise you will create a function that will show the user the total call of a simple interest loan. 
 #The formula for simple interest is Interest = principal x rate x time
-
+#just tesing my branching first
 
 
 #Prompt the user to enter their principal
-
+Principal = int(input("Please enter your Principal: "))
 
 #Prompt the user to enter their interest rate
-
+InterestRate = int(input("Please enter your Interest Rate: "))
 
 #Prompt the user to enter the time of their loan in years
+TimeofLoan = int(input("Enter the time of your loan in years: "))
+
+#calculate the total interest
+def TotalInterest():
+    Principal*InterestRate*TimeofLoan
+    print (TotalInterest)
 
 
-#calculate the to total interest
+TotalInterest()
 
 
 #Calculate the total amount the user would pay
+
```

Comment:
What about this one. Total amount would be total interest + principal

something like: total_amount = principal + total_interest()